src/games/planetwars/visualizer/visualsHelper.js

```diff
@@ -0,0 +1,477 @@
+const d3 = require('d3');
+const Config = require('./config');
+const Utils = require('./util');
+const space_math = Utils.SpaceMath;
+
+class Helper {
+  static attachToAllChildren(d3selector) {
+    return d3selector.data((d, i) => {
+      return Array(d3selector._groups[i].length).fill(d);
+    });
+  }
+
+  static translation(point) {
+    return 'translate(' + point.x + ',' + point.y + ')';
+  }
+
+  static rotate(angle) {
+    return 'rotate(' + angle + ')';
+  }
+
+  static resize(planet, amount) {
+    var tx = -planet.x * (amount - 1);
+    var ty = -planet.y * (amount - 1);
+    return Helper.translation({
+      x: tx,
+      y: ty
+    }) + ' scale(' + amount + ')';
+  }
+
+  static visualOwnerName(name) {
+    if (name === null) return 'None';
+    else return name;
+  }
+}
+
+// Helper objects start here
+
+class Planets {
```

Comment:
There's something about passing all these arguments to all these functions that makes it dirty. I don't know any certain solutions. Maybe wrap all possible needed input (selector, color_map, speed, size, scale, etc.) in some object?

Reply:
I'll just get rid of these static objects and change 'game' to have 1 of  each visual object.